quick-weblog.js

```diff
@@ -0,0 +1,70 @@
+function getArticle(url, api_key) {
```

Comment:
`getArticle` should be renamed or refactored as it is currently also updating the state of the page.